### PR TITLE
[IDEA][BLAZOR] Editable user control should stop updating value when focused and value binding should be switched to 'Edit' property of primitive type

### LINKED ISSUE #9: [IDEA][BLAZOR] Editable user control should stop updating value when focused and value binding should be switched to 'Edit' property of primitive type

### PR DESCRIPTION
**Pull request by PTKu**
closes #9

### CODE REVIEW

**Comment by Specter-13** [comment]
@PTKu there are some tests fails, probably because  of our change of **Edit** property in **OnlinerBase`1.cs** file. Can you please look at it?

File: src/ix.blazor/src/Ix.Presentation.Blazor.Controls/RenderableContent/RenderableComponentBase.cs [hunk 1]
```diff
@@ -20,6 +20,10 @@ namespace Ix.Presentation.Blazor.Controls.RenderableContent
     /// </summary>
     public partial class RenderableComponentBase : ComponentBase, IRenderableComponent
     {
+
+        public bool IsFocus { get; set; }
```
Comment (PTKu) [inline src/ix.blazor/src/Ix.Presentation.Blazor.Controls/RenderableContent/RenderableComponentBase.cs]: IsFocus -> HasFocus?

File: src/ix.connectors/src/Ix.Connector/ValueTypes/Onlines/OnlinerBase`1.cs [hunk 2]
```diff
@@ -535,12 +536,12 @@ protected virtual void WriteItem()
...
-            Parent.GetConnector()?.Logger.Verbose($"Value change; online; {Symbol};{origin};{newValue}'.");
-            EditValueChange?.Invoke(this, origin, newValue);
-            NotifyPropertyChanged(nameof(Edit));
-        }
+        //if (origin == null || !origin.Equals(newValue))
+        //{
+        //    Parent.GetConnector()?.Logger.Verbose($"Value change; online; {Symbol};{origin};{newValue}'.");
+        //    EditValueChange?.Invoke(this, origin, newValue);
+        //    NotifyPropertyChanged(nameof(Edit));
+        //}
```
Comment (PTKu) [inline src/ix.connectors/src/Ix.Connector/ValueTypes/Onlines/OnlinerBase`1.cs]: We will need to uncomment this... commented when debugging.

**Comment by PTKu** [comment]
> @PTKu there are some tests fails, probably because of our change of **Edit** property in **OnlinerBase`1.cs** file. Can you please look at it?

Yes, that's good, I've see that... we must uncomment the EditValue it is highlighted in the comments int this PR

**Pull request merged**